src/util/fileSizeUtil.ts

```diff
@@ -0,0 +1,11 @@
+const measurements = ['B', 'KB', 'MB', 'GB', 'TB'];
+
+export const bytesToSensibleFormat = (numberValue: number, measurement?: string): string => {
```

Comment:
Kanskje kalle denne variabelen for `bytes`? `numberValue` er veeeldig generisk og lite beskrivende

Reply:
Det starter som bytes, men vil jo mest sannsynligvis ende opp som en annen måleenhet. Enig i at det er generisk, men usikker på hva annet den kan være.